sktime/forecasting/base/adapters/_lagllama.py

```diff
@@ -0,0 +1,626 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Extension template for forecasters.
```

Comment:
Please remove all stuff that is related to the extension template or replace it. E.g., the docstring of this module should be about lag llama and not about the template

Reply:
Sorry about those. I was meaning to get rid of them and it skipped my mind once I got into the implementation. Just did get rid of them though! (except some in `get_test_params` which I am working on defining

Reply:
No problem, this happens sometime :)